README.md

```diff
@@ -49,17 +49,38 @@ Under virtual env:
 
     export DEBUG=1
 
+Postgres + PostGIS setup (OS X):
```

Comment:
Sure! Thats for previous step (django cities) I can move it to separate commit / PR if needed.